libs/payments/ui/src/lib/actions/determineLocation.ts

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use server';
+
+import { getApp } from '../nestapp/app';
+
+export const determineLocation = (
+  sanctionedLocations: string,
+  offeringId: string,
+  acceptLanguage?: string | null,
+  selectedLanguage?: string,
+  countryCode?: string
```

Comment:
This seems like more of a `getProductAvailabilityForLocation` rather than `determineLocation` -- `determineLocation` sounds like it's returning the user's location.